src/client/driver/index.js

```diff
@@ -10,6 +10,7 @@ import * as browser from '../browser';
 import executeActionCommand from './command-executors/execute-action-command';
 import executeWaitForElementCommand from './command-executors/execute-wait-for-element-command';
 import executeNavigateToCommand from './command-executors/execute-navigate-to-command';
+import executePendingBrowserWindowCommand from './command-executors/execute-pending-browser-window-command';
```

Comment:
So, as far as I understand @AlexanderMoskovkin, there is a pending command
`PendingBrowserWindowCommand`
We have a command that executes it
`Execute[PendingBrowserWindowCommand]Command`
And we need to name a function that executes that command
`Execute[Execute[PendingBrowserWindowCommand]Command]`

First, let's get rid of duplications
`Execute[Run[PendingBrowserWindowTask]Command]`
Isn't it still to complicated?